pilot/lib/pilot/build_manager.rb

```diff
@@ -38,7 +36,11 @@ def list(options)
         config[:app_identifier] = ask("App Identifier: ")
       end
 
-      builds = app.all_processing_builds + app.builds
+      if config[:app_platform].to_s.length == 0
+        config[:app_platform] = ask("App Platform (ios, appletvos, osx): ")
```

Comment:
Should we be doing validation on the return value of ask here? (make sure it is a valid platform, like in pilot/lib/pilot/options.rb, line 29)